tools/topology/sof-ehl-rt5660.m4

```diff
@@ -0,0 +1,193 @@
+#
+# Topology for ELKHARTLAKE with rt5660 codec
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Elkhartlake DSP configuration
+include(`platform/intel/ehl.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 ----> volume -----> SSP0 BE dailink 0
+# PCM1 <---- volume <----- DMIC48k (dmic48k, BE dailink 1)
+
+ifelse(HDMI, `1',
+`
+# PCM2 ----> volume -----> iDisp1 (HDMI/DP playback, BE link 5)
+# PCM3 ----> volume -----> iDisp2 (HDMI/DP playback, BE link 6)
+# PCM4 ----> volume -----> iDisp3 (HDMI/DP playback, BE link 7)
+# PCM5 ----> volume -----> iDisp3 (HDMI/DP playback, BE link 8)
+')
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     frames, deadline, priority, core)
```

Comment:
please update this commented macro, the last parameters are not shown

Reply:
fixed